### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by IshamMohamed**
Corrected some spellings

### CODE REVIEW

**Comment by IshamMohamed** [comment]
Generally others are fine.

**Comment by edgarcosta** [comment]
you added a typo and an unnecessary comma....

File: index.html [hunk 1]
```diff
@@ -275,7 +275,7 @@ <h3 class="firsttitle">Do we now need WPA3?</h3>
...
-			<p>Our main attack is against the 4-way handshake, and does not exploit access points, but instead targets clients. So it might be that your router does not require security updates. We strongly advise you to contact your vendor for more details. In general though, you can try to mitigate attacks against routers and access points by disabling client functionality (which is for example used in repeater modes) and disabling 802.11r (fast roaming). For ordinary home users, your priority should be updating clients such as laptops and smartphones.</p>
+			<p>Our main attack is against the 4-way handshake, and does not exploit access points, but instead targets clients. So, it might be that your router does not require security updates. We strongly advise you to contact your vendor for more details. In general thought, you can try to mitigate attacks against routers and access points by disabling client functionality (which is for example used in repeater modes) and disabling 802.11r (fast roaming). For ordinary home users, your priority should be updating clients such as laptops and smartphones.</p>
```
Comment (deed02392) [inline index.html:278]: > In general thought,

Typo - 'though' is the correct word choice.

**Comment by vanhoefm** [comment]
I don't see the spelling mistake? Combination of commits only adds a comma.

**Comment by edgarcosta** [comment]
@vanhoefm, @IshamMohamed added an unnecessary comma after the "So" and the replaced a correct "In general though, " by a "In general thought, "....

I would just close the PR.

**Pull request closed without merging**